tests/helper/tests/sysctl.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def check_sysctl(client, key, operator, value):
+    (exit_code, output, error) = client.execute_command("sysctl -a", quiet=True)
```

Comment:
sysctl binary is in `usr/sbin/sysctl`

From platform test run 
```
FAILED gardener/test/test_sysctls.py::test_sysctls[net.ipv4.conf.all.rp_filter-isnotornotset-1] - AssertionError: no error='bash: line 1: sysctl: command not found\n' expected
FAILED gardener/test/test_sysctls.py::test_sysctls[net.ipv4.conf.default.rp_filter-isnotornotset-1] - AssertionError: no error='bash: line 1: sysctl: command not found\n' expected
```

Reply:
Thanks for checking, I approved without checking if those tests were actually running. Should have been more careful.